.azure/publish.yml

```diff
@@ -0,0 +1,41 @@
+trigger:
+  tags:
+    include:
+      - "*"
+
+pool:
+  vmImage: "ubuntu-latest"
+
+jobs:
+  - job: "Publish"
+    pool:
+      vmImage: "ubuntu-latest"
+
+    steps:
+      - task: UsePythonVersion@0
```

Comment:
You need to run all this in the `generation/` folder. 

Also, is there an ESRP flow for Swift?

Reply:
You're right, we don't need do this.